services/clientlog/pkg/service/service_test.go

```diff
@@ -0,0 +1,3 @@
+package service_test
+
+// TODO: TEST!
```

Comment:
... ;)

Reply:
Not much business logic. Mocking services would be a nightmare. Removed the `_test.go` files